cudamapper/CMakeLists.txt

```diff
@@ -67,7 +67,15 @@ target_include_directories(${MODULE_NAME}
         $<INSTALL_INTERFACE:include>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
-target_link_libraries(${MODULE_NAME} gwbase gwio cub)
+
+# optionally link against htslib
+if (gw_build_htslib)
```

Comment:
let's check this against the availability of the `htslib` target. if we change the the htslib is built or accessed, then this part of the code won't have to change

Reply:
good idea. Fixed